### PR TITLE
Milestone2

### PR DESCRIPTION
**Pull request by addod19**
## In this milestone, we performed the following actions:
- Prepare directory structure
- Implement the App Component
- Implement BookList Component
- Implement Book Component
- Implement BooksForm Component
- Define Redux Actions and Reducers
- Initialize the Redux Store

Kindly review our project :man_technologist:

### CODE REVIEW

**Review by Torres-ssf — CHANGES_REQUESTED** [review]
### Changes Required :muscle: 
Hi team :hand: 
Excellent job on this milestone so far :+1: :clap: 
We only need to make some small change before moving on

- [x] This linter.yml file belongs to the javascript module configuration. We need to use the one from React module. Check [here](https://github.com/microverseinc/linters-config/blob/master/react-redux/.github/workflows/linters.yml).
https://github.com/evansinho/React-Bookstore/blob/c4f63649a9c02f95262197201d05761b48987d13/.github/workflows/linters.yml#L9-L22

Please kindly make these changes and submit to code review again.
Feel free to message me on slack if you have any questions :smile: 
Happy Coding :computer: 
**Sergio Torres**

File: src/components/Book.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    <td>
+      {' '}
+      {book.category}
+      {' '}
+    </td>
+  </tr>
+);
+
+Book.propTypes = {
+  book: PropTypes.func.isRequired,
```
Comment (Torres-ssf) [inline src/components/Book.js]: - [ ] Why `book` is defined as a `func` here? Isn't supposed to be an object?
Screenshot from 2020-08-10 14-52-08

**Review by Torres-ssf — CHANGES_REQUESTED** [review]
### Changes Required :muscle:

File: src/components/Book.js (hunk 1 above)
Comment (Oghenebrume50) [reply src/components/Book.js]: Hi @evansinho @addod19 following this review, I see that the change for this was not made, is there a reason why Book is being checked as a func and not an object? Please leave or comment or change it if it was an oversight :slightly_smiling_face:

**Review by Oghenebrume50 — CHANGES_REQUESTED** [review]
### CHANGES REQUIRED

Hi @addod19 @evansinho  this looks good, but I think you guys left a correction from the last review undone, please find comment below and work on it :slightly_smiling_face: 

**Please make changes and request for another review** :slightly_smiling_face: 

**RAPHAEL NORIODE** :mask: stay_safe

**Review by bellom — APPROVED** [review]
## Hi Team :clap: 

**Great work making all the necessary corrections from the previous review.**

## Congratulation :+1:  
_Go ahead and merge to dev and don't forget to submit project completion form.
Good luck in the next milestone :slightly_smiling_face:_ 

**Best,**

**Oluwaseun Bello**
**_Slack Handle - @sean bellom_**

**Pull request merged**